integrated_channels/sap_success_factors/models.py

```diff
@@ -143,3 +155,74 @@ def __repr__(self):
         Return uniquely identifying string representation.
         """
         return self.__str__()
+
+
+class SAPSuccessFactorsLearnerCompletionPayload(object):
```

Comment:
This is a pattern that I've seen in various locations but unfortunately we're not very consistent with it in Open edX -- sometimes there is an FK and sometimes the data is replicated into the audit record with no actual FK defined.  Does anyone feel strongly one way or the other?